backend/qemu.pm

```diff
@@ -872,7 +872,15 @@ sub start_qemu ($self) {
         }
 
         sp('device', 'usb-kbd') if $use_usb_kbd;
-        sp('smp', $vars->{QEMUTHREADS} ? [qv "$vars->{QEMUCPUS} threads=$vars->{QEMUTHREADS}"] : $vars->{QEMUCPUS});
+
+        my $smp_config = [$vars->{QEMUCPUS}];
+
+        for my $qkey (qw(sockets dies clusters cores threads)) {
```

Comment:
Not sure what others prefer, but I like when I can grep the code for a var name directly.
So writing `qw(QEMUSOCKETS QEMUDIES QEMUCLUSTERS QEMYCORES QEMUTHREADS)` is what I would do here.

Reply:
But `$qkey` is also used as-is. Converting it the other way around is likely more annoying. So for me it looks acceptable as it is.

Reply:
oh, you're right. I missed that one.

Reply:
Changed it anyway, being able to grep for the variables is good.